tmol/tests/pose/test_pose_stack_construction_benchmark.py

```diff
@@ -1,19 +1,71 @@
 import pytest
 
 # from tmol.score.score_types import ScoreType
-from tmol.pose.pose_stack import PoseStack
+from tmol.pose.pose_stack_builder import PoseStackBuilder
+from tmol.tests.torch import zero_padded_counts
 
 
 @pytest.mark.parametrize("n_poses", [1, 3, 10, 30, 100])
-@pytest.mark.benchmark(group="pose_stack_construction")
-def test_pose_construction_benchmark(
+@pytest.mark.benchmark(group="pose_stack_construction_from_residues")
+def test_pose_construction_benchmark_from_residues(
     benchmark, n_poses, rts_ubq_res, default_database, torch_device
 ):
-    pose_stack1 = PoseStack.one_structure_from_polymeric_residues(
+    pose_stack1 = PoseStackBuilder.one_structure_from_polymeric_residues(
         rts_ubq_res, torch_device
     )
 
     @benchmark
     def construct_pass():
-        pose_stack_n = PoseStack.from_poses([pose_stack1] * n_poses, torch_device)
+        pose_stack_n = PoseStackBuilder.from_poses(
+            [pose_stack1] * n_poses, torch_device
+        )
+        return pose_stack_n
+
+
+@pytest.mark.parametrize("n_poses", zero_padded_counts([1, 3, 10, 30, 100]))
+@pytest.mark.benchmark(group="pose_stack_construction_from_seq")
```

Comment:
since it's just pose from sequence can this be tested with variable lengths & stack sizes?  Just `"ACDEFGHIKLMNPQRSTVWY" * nrepeats`?